starter-core/src/main/java/io/micronaut/starter/feature/dependencies/GradleConfiguration.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.starter.feature.dependencies;
+
+public enum GradleConfiguration {
```

Comment:
Using an enum would prevent in the future using custom configurations (eg `developmentOnly`, `documentation`, etc).

Also, `annotationProcessor` is missing.

Reply:
> Also, annotationProcessor is missing.

I've added annotationProcessor